.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: go
+
+go:
+  - 1.13.x
```

Comment:
```suggestion
  - 1.14.x
```

Reply:
Why shall we upgrade to major version ? I've experienced few issues with v14 in other projects, have you tried that ??

Reply:
ethermint and the SDK are currently using 1.14. They work fine

Reply:
done